docs/readthedocs/source/doc/Nano/Overview/install.md

```diff
@@ -40,4 +71,4 @@ In a conda environment, `source bigdl-nano-init` will also be added to `$CONDA_P
 
```

Comment:
The expression on line 70 is not very precise, `source bigdl-nano-init` won't be added to `$CONDA_PREFIX/etc/conda/activate.d/` until users run `source bigdl-nano-init` manually, We can point this out here or in the comments on line 64

Reply:
others LGTM

Reply:
Done, please take a look at  some changes about cmd.